src/components/Navbar/index.tsx

```diff
@@ -18,32 +18,27 @@ const { white, blue, gray } = uiColors;
 export const Navbar: React.FC = () => {
   const { isAuthenticated } = useAuthStateContext();
   const legacyURL = useLegacyUIURL();
-  const navbarAnalytics = useNavbarAnalytics();
+  const uiURL = getUiUrl();
 
   if (!isAuthenticated) {
     return null;
   }
+
   return (
     <StyledHeader>
       <InnerWrapper>
-        <Link
-          to={routes.myPatches}
-          onClick={() => navbarAnalytics.sendEvent({ name: "Click Logo Link" })}
-        >
-          <Logo>
-            <EvergreenLogo />
-            <StyledSubtitle>Evergreen</StyledSubtitle>
-          </Logo>
-        </Link>
+        <NavActionContainer>
```

Comment:
Was taking out the analytics intentional?

Reply:
no it wasn't must have been a bad merge